Sources/TensorFlow/Operators/Image.swift

```diff
@@ -0,0 +1,331 @@
+  var images = images
+  let singleImage = images.rank == 3
+  if singleImage {
+    images = images.rankLifted()
+  }
+  let size = Tensor([Int32(size.0), Int32(size.1)])
+  let scale = Tensor<Float>(size) / Tensor<Float>([Float(images.shape[1]), Float(images.shape[2])])
+  switch method {
+  case .nearest:
+    images = resizeNearestNeighbor(
+      images: images,
+      size: size,
+      halfPixelCenters: true)
+  case .bilinear:
+    if antialias {
+      images = scaleAndTranslate(
+        images: images,
+        size: size,
+        scale: scale,
+        translation: Tensor(zeros: [2]),
+        kernelType: "triangle")
+    } else {
+      images = resizeBilinear(
+        images: images,
+        size: size,
+        halfPixelCenters: true)
+    }
+  case .bicubic:
+    if antialias {
+      images = scaleAndTranslate(
+        images: images,
+        size: size,
+        scale: scale,
+        translation: Tensor(zeros: [2]),
+        kernelType: "keyscubic")
+    } else {
+      images = resizeBicubic(
+        images: images,
+        size: size,
+        halfPixelCenters: true)
+    }
+  case .lanczos3:
+    images = scaleAndTranslate(
+      images: images,
+      size: size,
+      scale: scale,
+      translation: Tensor(zeros: [2]),
+      kernelType: "lanczos3",
+      antialias: antialias)
+  case .lanczos5:
+    images = scaleAndTranslate(
+      images: images,
+      size: size,
+      scale: scale,
+      translation: Tensor(zeros: [2]),
+      kernelType: "lanczos5",
+      antialias: antialias)
+  case .gaussian:
+    images = scaleAndTranslate(
+      images: images,
+      size: size,
+      scale: scale,
+      translation: Tensor(zeros: [2]),
+      kernelType: "gaussian",
+      antialias: antialias)
+  case .mitchellcubic:
+    images = scaleAndTranslate(
+      images: images,
+      size: size,
+      scale: scale,
+      translation: Tensor(zeros: [2]),
+      kernelType: "mitchellcubic",
+      antialias: antialias)
+  }
+
+  if singleImage {
+    images = images.squeezingShape(at: 0)
+  }
+
+  return images
+}
+
+/// Resize images to size using area interpolation.
+///
+/// - Parameters:
+///   - images: 4-D Tensor of shape [batch, height, width, channels] or 3-D Tensor of shape [height, width, channels].
+///   - size: A tuple of two Int: (new_height, new_width). The new size for the images.
+/// - Precondition: The images must have rank 3 or 4.
+/// - Precondition: The size must be positive.
+@inlinable
+public func resizeArea<Scalar: TensorFlowNumeric>(
+  images: Tensor<Scalar>,
+  size: (Int, Int),
+  alignCorners: Bool = false
+) -> Tensor<Float> {
+  precondition(
+    images.rank == 3 || images.rank == 4,
+    "The images tensor must have rank 3 or 4.")
+  precondition(size.0 > 0 && size.1 > 0, "The size must be positive.")
+  var images = images
+  let singleImage = images.rank == 3
+  if singleImage {
+    images = images.rankLifted()
+  }
+  let size = Tensor([Int32(size.0), Int32(size.1)])
+
```

Comment:
Minor: please delete extra whitespace. For function bodies that aren't too dense, I personally prefer having no empty lines in the body, unless the empty lines clearly separate logical stages of execution.